20-packages.ks

```diff
@@ -34,7 +34,7 @@ OpenIPMI
 OpenIPMI-tools
 openssl
 foreman-proxy
```

Comment:
Does this also need to be `tfm-foreman-proxy`?

Reply:
no, the package name is the same